### PR TITLE
[chibios] Add PX4FMU V4 (pixracer)

### PR DESCRIPTION
**Pull request by fvantienen**
This adds support for the Pixracer (PX4FMU) using chibios.

### CODE REVIEW

File: conf/boards/px4fmu_4.0_chibios.makefile [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+SBUS_PORT ?= UART6
+RADIO_CONTROL_SPEKTRUM_PRIMARY_PORT   ?= UART6
+
+#The modem serial on px4 is called serial 1, but connected to uart2 on the f4
+MODEM_PORT ?= UART2
+MODEM_BAUD ?= B57600
+
+#The GPS serial on px4 is called serial 3, but connected to uart4 on the f4
+GPS_PORT ?= UART4
+GPS_BAUD ?= B38400
```
Comment (gautierhattenberger) [inline conf/boards/px4fmu_4.0_chibios.makefile]: maybe we can set a default speed of 57600 for recent board

File: sw/airborne/boards/px4fmu/chibios/v4.0/board.c [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+
+  stm32_gpio_init();
+  stm32_clock_init();
+}
+
+#if HAL_USE_SDC || defined(__DOXYGEN__)
+/**
+ * @brief   SDC card detection.
+ */
+bool sdc_lld_is_card_inserted(SDCDriver *sdcp) {
```
Comment (gautierhattenberger) [inline sw/airborne/boards/px4fmu/chibios/v4.0/board.c]: if you have a SD card on this board, you probably need to implement this function like: https://github.com/paparazzi/paparazzi/blob/master/sw/airborne/boards/chimera/chibios/v1.0/board.c#L220

Comment (fvantienen) [reply sw/airborne/boards/px4fmu/chibios/v4.0/board.c]: Yes I thought so as well, but these lines aren't connected on the pixracer so this is not possible. I looked at ardupilot and they have it like this and it works as well.

File: conf/boards/px4fmu_4.0_chibios.makefile (hunk 1 above)
Comment (fvantienen) [reply conf/boards/px4fmu_4.0_chibios.makefile]: I will change this

**Pull request merged**